src/config/load.ts

```diff
@@ -0,0 +1,32 @@
+import path from 'path';
+
+import { register } from 'esbuild-register/dist/node';
+
+import { type SkubaConfig, skubaConfigSchema } from './types';
+
+import { getDestinationManifest } from 'cli/configure/analysis/package';
+import { log } from 'utils/logging';
+
+const CONFIG_FILENAME = 'skuba.config.ts';
+
+export const loadSkubaConfig = async (cwd?: string): Promise<SkubaConfig> => {
+  // FIXME: remove underlying `process.exit`
+  const manifest = await getDestinationManifest({ cwd });
+
+  const configPath = path.join(manifest.path, '..', CONFIG_FILENAME);
+
+  const esbuildRegistration = register({ target: 'node16' });
```

Comment:
It might be worth having a hot reloading ability when the config changes, e.g. restart the server started with `skuba start` or anything like that.

For reference, we do this in Crackle: 
https://github.com/seek-oss/crackle/blob/49b48ee405477cccdc1d6e7bbbb4c26b921c7760/packages/cli/src/index.ts#L33-L44